src/languages/es.js

```diff
@@ -552,12 +559,27 @@ export default {
         error: {
             firstNameLength: 'El nombre no debe tener más de 50 caracteres',
             lastNameLength: 'El apellido no debe tener más de 50 caracteres',
-            characterLimit: ({limit}) => `Supera el límite de ${limit} caracteres`,
             hasInvalidCharacter: ({invalidCharacter}) => `Por favor elimina ${invalidCharacter} del campo nombre.`,
             comma: 'la coma',
             semicolon: 'el punto y coma',
         },
     },
+    privatePersonalDetails: {
+        personalDetails: 'Datos personales',
+        privateDataMessage: 'Estos detalles se utilizan para viajes y pagos. Nunca se mostrarán en su perfil público.',
+        legalName: 'Nombre completo',
+        legalFirstName: 'Nombre',
+        legalLastName: 'Apellidos',
+        homeAddress: 'Domicilio',
+        error: {
+            hasInvalidCharacter: '',
+            legalFirstNameEmpty: '',
+            legalLastNameEmpty: '',
```

Comment:
Schmuck... I'm going to deal with cleaning up some of these error messages tomorrow